spec/presenters/service_standard_presenter_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe ServiceStandardPresenter, "#content_id" do
+  it "is a preassigned UUID" do
```

Comment:
mix of `is a` and `returns a` for methods that just 'return a thing', do we have a standard way of describing return values? (Bordering on pedantry? 😛)